src/mavedb/lib/score_sets.py

```diff
@@ -63,7 +64,12 @@ def search_score_sets(db: Session, owner: Optional[User], search: ScoreSetsSearc
     query = query.filter(~ScoreSet.superseding_score_set.has())
 
     if owner is not None:
```

Comment:
@bencap wrote:

> Same as in lib/experiments.py:
> 
> Maybe requester is a better descriptor for this argument now? I got kind of confused on 63 when we do ScoreSet.contributors.has(Contributor.orcid_id == owner.username), since it reads very similarly to 62.